src/Uno.Foundation/Uno.Core.Extensions/Uno.Core.Extensions.Compatibility/Reflection/TypeExtensions.cs

```diff
@@ -187,5 +189,37 @@ public static bool IsNullable(this Type type)
 				|| Nullable.GetUnderlyingType(type) != null;    // is Nullable<T>
 		}
 #endif
+
+		public static bool IsGenericDescentOf(this Type type, Type genericTypeDefinition)
```

Comment:
Please add comment on those kind of method. For instance should the `genericTypeDefinition` has to be 'constructed' (having arguments for the generic parameters)?